nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -156,6 +152,9 @@ EXEMPT_VIEW_PERMISSIONS = [
     # 'ipam.prefix',
 ]
 
+# If hosting Nautobot in a subdirectory, you must set this value to match the base URL prefix configured in your HTTP server (e.g. `/nautobot`). When not set, URLs will default to being prefixed by `/`.
```

Comment:
Still learning how this works. If the HTTP server is setting `SCRIPT_NAME` correctly, then `FORCE_SCRIPT_NAME` is not required, correct? This would only be needed if `SCRIPT_NAME` is incorrect?

Reply:
MOST of the application works correctly without deliberately setting `FORCE_SCRIPT_NAME`, but there are a few edge cases mostly related to not being able to leverage `django.urls.reverse()` consistently due to circular imports caused by having to end up loading `nautobot.core.urls` where nearly every single model in the project is loaded to facilitate `nautobot.core.views.HomeView` and `SearchView`. 

So for now, to allow for places where `APISelect` widgets are being used, namely the `Device.position` dynamic lookup, this is the only way to assert that we can reliably make the app receive the correct `SCRIPT_NAME`. If we can refactor enough to be able to use `reverse()` reliably, we can stop hard-coding API URLs in the code and can simplify deployment even further.